src/renderers/dom/ReactDOMNodeStreamRenderer.js

```diff
@@ -19,10 +19,8 @@ var Readable = require('stream').Readable;
 
 // This is a Readable Node.js stream which wraps the ReactDOMPartialRenderer.
 class ReactMarkupReadableStream extends Readable {
-  constructor(element, makeStaticMarkup) {
-    // Calls the stream.Readable(options) constructor. Consider exposing built-in
-    // features like highWaterMark in the future.
-    super({});
+  constructor(element, makeStaticMarkup, {highWaterMark} = {}) {
```

Comment:
Any specific reason we're destructing in these constructors? It doesn't look like `highWaterMark` is used here, so maybe we can just do:

```js
constructor(element, makeStaticMarkup, options = {}) {
   super(options);
   // ...
}